### PR TITLE
Added landing page message #1

### PR DESCRIPTION
**Pull request by codrAshraf**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>My Potionfy</title>
+    <title>Potionfy | Landing page</title>
   </head>
   <body>
     
-    <h1>Potionfy</h1>
+    <h1>Welcome to Potionfy! Enjoy your stay</h1>
```
Comment (ashraf267) [inline index.html:8]: Could you remove the "Enjoy your stay" string and make the "Welcome to Potionfy!" string red?

Comment (codrAshraf) [reply index.html:8]: Sure, can do!

**Review by ashraf267 — CHANGES_REQUESTED** [review]
Please, change the codes as per my comment on the review.

**Review by ashraf267 — APPROVED** [review]
Nicely done, thanks.

**Pull request merged**